

semantic_camera_img_input_topic: "/unity_ros/Quadrotor/Sensors/SemanticCamera/image_raw"
semantic_camera_info_input_topic: "/unity_ros/Quadrotor/Sensors/SemanticCamera/camera_info"
 
pcl_input_topic: "/perception/pcl/out_colored"

rgb_img_input_topic: "/realsense/rgb/left/image_raw"

publish_2d_processed_semantic_img: false
2d_processed_semantic_img_out_topic: "/perception/processed_semantic_image"

publish_overlayed_rgb_img: false
overlayed_rgb_img_out_topic: "/perception/overlayed_rgb_image"

publish_object_pcl: true
object_pcl_out_topic: "/perception/object_pcl"

publish_object_bb: true
object_bb_out_topic: "/perception/objects_bb"

publish_object_vis_markers: true
object_vis_out_topic: "/perception/object_marker"

#filtering params
mean_k: 20
thresh: 0.1


